packages/components/src/List/ListComposition/SortBy/SortBy.component.js

```diff
@@ -3,11 +3,17 @@ import PropTypes from 'prop-types';
 import { Navbar, NavDropdown, Nav, NavItem, MenuItem } from 'react-bootstrap';
 import uuid from 'uuid';
 
+import { DISPLAY_MODE } from '../constants';
 import { useListContext } from '../context';
 
 function SortBy(props) {
 	const { id, initialValue, options, onChange, value } = props;
-	const { sortParams, setSortParams, t } = useListContext();
+	const { displayMode, sortParams, setSortParams, t } = useListContext();
+
+	if (displayMode === DISPLAY_MODE.TABLE) {
```

Comment:
You break the rules of hooks.
Move that after the useEffect please

Reply:
done